rotkehlchen/chain/ethereum/modules/curve/pools_cache.py

```diff
@@ -203,11 +228,12 @@ def save_curve_pools_to_cache(
             key_parts=[GeneralCacheType.CURVE_POOL_ADDRESS, lp_token_address],
             values=[pool_address],
         )
-        globaldb_set_general_cache_values(
-            write_cursor=write_cursor,
-            key_parts=[GeneralCacheType.CURVE_POOL_TOKENS, pool_address],
-            values=coins,
-        )
+        for i, coin in enumerate(coins):
+            globaldb_set_general_cache_values(
```

Comment:
I believe we tend to avoid single letter variables, can you rename to `index` or `coin_index`?